### PR TITLE
Additive closures Documentations + 3 functors

### PR DESCRIPTION
**Pull request by kamalsaleh**
1- For Ab-category C construct the inclusion functor C --> AdditiveClosure(C).
2- For  F:C-->D construct G: AdditiveClosure(C) --> AdditiveClosure(D).
3- For  F:C-->D with additive D construct G: AdditiveClosure(C) --> D.

### CODE REVIEW

**Comment by kamalsaleh** [comment]
Hallo Sebastian, the PR in its current status is complete and ready for review. If I needed other operations, I would do that in another PR.

File: FreydCategoriesForCAP/gap/AdditiveClosure.gd [hunk 1]
```diff
@@ -14,57 +14,141 @@
...
 
+#! @Description
+#! The input is an object $A$ in an Ab-category $C$. The output is the image of $A$ under the inclusion functor $\iota:C\to C^\oplus$.
+#! @Arguments A
+#! @Returns a CapCategoryObject
 DeclareAttribute( "AsAdditiveClosureObject",
                   IsCapCategoryObject );
 
+#! @Description
+#! The input is a formal direct sums $A=A_1\oplus\dots\oplus A_m$, $B=B_1\oplus\dots\oplus B_n$ and
```
Comment (sebastianpos) [inline FreydCategoriesForCAP/gap/AdditiveClosure.gd]: plural needed

File: FreydCategoriesForCAP/gap/AdditiveClosure.gd [hunk 2]
```diff
@@ -14,57 +14,141 @@
...
+
 ####################################
 ##
 #! @Section Attributes
 ##
 ####################################
 
+#! @Description
+#! The input is some additive closure category $C^\oplus$. The output is $C$.
+#! @Arguments C^\oplus
```
Comment (sebastianpos) [inline FreydCategoriesForCAP/gap/AdditiveClosure.gd]: This is latex code.

**Comment by kamalsaleh** [comment]
I have updated the documentation in this commit https://github.com/homalg-project/CAP_project/pull/415/commits/7a50def07aaffd073accd2c14632391ab25c7334

**Pull request merged**